src/components/Authz/Credentials.js

```diff
@@ -1,167 +1,71 @@
 import React, { useState, useEffect } from 'react';
-import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
-
 import { LinkWithAuthz } from './LinkWithAuthz';
+import { _matchCredentialId } from '../../utils/retrieveCredentialsInfo';
 
 import './Credentials.scss';
 
-export function Credentials({ ...props }) {
-
+export function Credentials({  app, credentials_ids, apiPath = 'seacat-auth', cleanupTime = 1000 * 60 * 60 * 24 }) {
```

Comment:
I dont hink we need to specify `apiPath` - it will likely be always `seacat-auth`

Reply:
@Pe5h4 I brought it up. You told me to keep this prop and not to remove it
If we agree to remove it. I'll do it.

Reply:
Ok, it wont be the case I suppose. What can be done is that you will always need to pass the proxied API, so e.g.

this is what you will do in parent:

```
const SeaCatAuthAPI = app.axiosCreate('seacat-auth');

...

return <Credentials app={app} API={SeaCatAuthAPI} ... />
```

THen in Credentials component 

```
export function Credentials({ app, credentials_ids, API }) { ... }
```

However this would be a breaking change and you would really need to state it in changelog and do the appropriate change wherever Credentials component is used PRIOR merging this (or it should be merged simultaneously)

Reply:
ofcourse you would need to update the readme/description as well. but take this as another idea how to handle this. Since what I remember, Ales told me that we should stripe any mention of seacat or logman from our components and shell libraries.